spec/features/web_intake/routing_spec.rb

```diff
@@ -84,6 +86,39 @@
     fill_in "ZIP code", with: zip_code
     click_on "Continue"
 
-    expect(page.html).to have_text "Our team at Other Location is here to help!"
+    expect(page.html).to have_text "Our team at Diagon Alley is here to help!"
+  end
+
+  scenario "routing by state" do
+    visit "/questions/welcome"
+    click_on "File taxes with help"
+
+    expect(page).to have_text "File with the help of a tax expert!"
+    click_on "Continue"
+
+    expect(page).to have_text "What years do you need to file for?"
+    check "2020"
+    click_on "Continue"
+
+    expect(Intake.last.source).to eq nil
+    expect(page).to have_text "Thanks for visiting the GetYourRefund demo application!"
+    click_on "Continue to example"
+
+    expect(page).to have_text "Let's get started"
+    click_on "Continue"
+
+    expect(page).to have_text "Let’s check a few things"
+    check "None of the above"
+    click_on "Continue"
+
+    expect(page).to have_text "Just a few simple steps to file!"
+    click_on "Continue"
+
+    expect(page).to have_text "let's get some basic information"
+    fill_in "Preferred name", with: "Luna Lovegood"
+    fill_in "ZIP code", with: "28806"
+    click_on "Continue"
```

Comment:
If at this point you want to check the routing params, that might make sense to assert it's routing by state.